cms/envs/common.py

```diff
@@ -83,7 +83,7 @@
 
     # If set to True, Studio won't restrict the set of advanced components
     # to just those pre-approved by edX
-    'ALLOW_ALL_ADVANCED_COMPONENTS': False,
+    'ALLOW_ALL_ADVANCED_COMPONENTS': True,
```

Comment:
Please remove this before merging -- we don't want to accidentally turn this on in prod :)